modules/identity-provider-add.adoc

```diff
@@ -12,10 +12,7 @@
 // * authentication/identity_providers/configuring-google-identity-provider.adoc
 // * authentication/identity_providers/configuring-oidc-identity-provider.adoc
 
-// OIDC, GitHub, and Google IDPs do not support username/password login commands
-ifeval::["{context}" == "configuring-oidc-identity-provider"]
-:no-username-password-login:
-endif::[]
+// GitHub and Google IDPs do not support username/password login commands
```

Comment:
some OIDC providers still won't allow password logins, not sure how to work that in

Reply:
Interesting. I'm wondering whether we should leave it as a token login for OIDC and then just say that some OIDC IDPs allow username/password logins. wdyt?

Reply:
@stlaz I switched back to using tokens for OIDC since not all support username/password login. I think that it's too deep for the product docs to point the spec you mentioned in slack, so for now it's just a generic note that some support it. If needed, I could always add for them to check out the docs for their IDP to find out one way or the other.